chia/cmds/passphrase_funcs.py

```diff
@@ -50,19 +51,50 @@ def tidy_passphrase(passphrase: str) -> str:
     return passphrase.strip()
 
 
-def prompt_for_new_passphrase() -> str:
+def prompt_to_save_passphrase() -> bool:
+    save: bool = False
+
+    try:
+        if supports_os_passphrase_storage():
+            location: Optional[str] = None
+
+            if sys.platform == "darwin":
+                location = "macOS Keychain"
+
+            if location is None:
+                raise ValueError("OS-specific credential store not specified")
+
+            print(
+                "\n"
+                "Your passphrase can be stored in your system's secure credential store. "
+                "Chia services will be able to access your keys without prompting for your passphrase."
+            )
+            save = prompt_yes_no(f"Would you like to save your passphrase to the {location} (y/n) ")
+
+    except Exception as e:
+        print(f"Caught exception: {e}")
+        return False
+
+    return save
+
+
+def prompt_for_new_passphrase() -> Tuple[str, bool]:
     min_length: int = Keychain.minimum_passphrase_length()
     if min_length > 0:
         n = min_length
         print(f"\nPassphrases must be {n} or more characters in length")  # lgtm [py/clear-text-logging-sensitive-data]
     while True:
-        passphrase = tidy_passphrase(getpass("New Passphrase: "))
-        confirmation = tidy_passphrase(getpass("Confirm Passphrase: "))
+        passphrase: str = tidy_passphrase(getpass("New Passphrase: "))
+        confirmation: str = tidy_passphrase(getpass("Confirm Passphrase: "))
```

Comment:
The original intent was to prevent an accidental/unintentional press of the spacebar (leading or trailing) from locking the user out of their keys, but that predated the use of a confirmation passphrase. I think it can be safely removed now. If someone wants to use an all-whitespace passphrase, well... more power to them.

Reply:
If I already have a space as the first or last character in my password, removing this function will make me unable to unlock my keys. I don't think there's a simple way to remove this, we could only remove it for new installs, or when someone changes their password (if I understand correctly)

Reply:
Passphrase functionality isn't enabled for anyone yet, so it should be fine to remove it.

Reply:
Also just to further explain, the tidy_passphrase function is only used when setting a passphrase, or when reading it in from a file (which is another case where trimming the whitespace could be useful, but probably not necessary).

Reply:
I see. I think passwords are already (most likely) really bad and low-entropy. It seems risky to lower it further by stripping whitespace.

Reply:
I'd probably remove this as well

Reply:
Removed